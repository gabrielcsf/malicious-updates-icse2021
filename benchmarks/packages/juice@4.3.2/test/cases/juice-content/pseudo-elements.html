<html><body>
<style>
a {
  text-decoration: underline;
}

a:before,
a:after {
  content: "a";
}

* a:before {
  content: 'Â®\+';
}

a:after,
a::after {
  font-weight: bold;
}

a:first-child {
  color: blue;
}

b:after {
 content: " ";
}

b:after {
 content: "b";
 color: green;
}

b :last-child {
  color: red;
}

c:not(.not) {
    color: red;
}

d:before {
  content: url(data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACwAAAAAAQABAAACAkQBADs=);
}

d:after {
  content: url('data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACwAAAAAAQABAAACAkQBADs=');
  width: 20%;
}
</style>
<a href="#">Test</a>
<b></b>
<b><span></span></b>
<c></c>
<c class="not"></c>
<d><span>test</span></d>
</body></html>
