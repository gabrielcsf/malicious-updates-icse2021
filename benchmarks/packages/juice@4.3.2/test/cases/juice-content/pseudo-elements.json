{
  "url": "./",
  "removeStyleTags": true,
  "inlinePseudoElements": true
}
