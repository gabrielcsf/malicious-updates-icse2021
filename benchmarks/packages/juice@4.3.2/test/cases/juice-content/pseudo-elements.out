<html><body>

<a href="#" style="text-decoration: underline; color: blue;"><span>Â®+</span>Test<span style="font-weight: bold;">a</span></a>
<b><span style="color: green;">b</span></b>
<b><span style="color: red;"></span><span style="color: green;">b</span></b>
<c style="color: red;"></c>
<c class="not"></c>
<d><img src="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACwAAAAAAQABAAACAkQBADs="><span>test</span><img style="width: 20%;" src="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACwAAAAAAQABAAACAkQBADs="></d>
</body></html>
